redcap/project.py

```diff
@@ -319,6 +321,10 @@ def export_records(self, records=None, fields=None, forms=None,
             export.
         filter_logic : string
             specify the filterLogic to be sent to the API.
+        date_begin : string
```

Comment:
Rather than parsing a string to a datetime object (requiring a dependency on `python-dateutil` and then reformatting it out to a string, these two parameters can be datetimes. So it's on the caller to make datetime objects, but I don't think that's too much to ask.

Reply:
Easy enough to change. Thanks for your reply.